### DISCUSSION TITLE
Unable to Use the latest version of PHP

### DISCUSSION THREAD

**Discussion by Tomrm96** [leokhoa/laragon#837] [Q&A] [answered]
![Screenshot 2024-07-19 113632](https://github.com/user-attachments/assets/768f26ea-a8b6-4e95-9013-35e85d08d65c)

Hi Team, 

Im trying to use the latest version of PHP, I extracted the Latest version into the correct folder and enables 7zip, I can create a project fine bt this error message keeps popping up?

Can anyone help?

**Comment by StenlyAndika** [comment, marked as answer]
apache needs to be updated to latest version

**Comment by luisAntonioLAGS** [comment]
> ![Screenshot 2024-07-19 113632](https://private-user-images.githubusercontent.com/100447956/350363440-768f26ea-a8b6-4e95-9013-35e85d08d65c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE0MTUzNTAsIm5iZiI6MTcyMTQxNTA1MCwicGF0aCI6Ii8xMDA0NDc5NTYvMzUwMzYzNDQwLTc2OGYyNmVhLWE4YjYtNGU5NS05MDEzLTM1ZTg1ZDA4ZDY1Yy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcxOVQxODUwNTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kOTVkM2ZjYzM3MTc2ZGIwNjNkZjNjMzQwMWJmYzU1OWM1YmI4MmE0NmFlZDYyYmViNjBjYTc0NTc5MzI2ZDYyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.6fyuoTFsLejkcbbiTL-7C7a0RVuUqzbHrDaNHWni6rw)
> 
> Hi Team,
> 
> Im trying to use the latest version of PHP, I extracted the Latest version into the correct folder and enables 7zip, I can create a project fine bt this error message keeps popping up?
> 
> Can anyone help?

Update apache version.
https://usersdrive.com/7xw0h6vttq07.html
OR
https://www.mediafire.com/file/okxm73kwohqspqj/httpd-2.4.61-240703-win64-VS17.zip/file

**Comment by Tomrm96** [comment]
Thanks all have a good weekend :)
